flexdata/data.py

```diff
@@ -435,14 +457,38 @@ def read_flexraylog(path, sample = 1):
     geom.parameters['det_pixel'] *= sample
     geom.parameters['img_pixel'] *= sample
 
-    # Some Flexray scanner-specific motor offset corrections:
-    _flex_motor_correct_(geom)
+    geom = correct_roi(geom)
+
+    if sample != 1:
+        msg = f"Adjusted geometry by binning by {sample}"
+        logging.info(msg)
+        geom.log(msg)
 
     return geom
 
-def read_flexraymeta(path, sample = 1):
+
+def read_flexraymeta(*args):
+   warnings.warn("""
+read_flexraymeta is depecrated.
+
+This function combined too much functionality. If you want similar functionality to what
+read_flexraymeta provided, use:
+>>> from flexdata import data
+>>> from flexdata import correct
+>>> geom = data.parse_flexraymeta(path, sample=binning)
+>>> geom = correct.correct(geom,
+                           profile='cwi-flexray-2019-05-24',
+                           do_print_changes=True)
+>>> geom = correct.correct_vol_center(geom)
+
+
+""", DeprecationWarning, stacklevel=2)
+   raise NotImplementedError()
```

Comment:
Indentation is off here.

Reply:
Will fix this later. It does not crash..